source/manual/debian-packaging.html.md

```diff
@@ -135,11 +135,75 @@ the recipe.
 #### Test the recipe
 
 - The Jenkins job will produce a `.deb` package in the `Build Artifacts`.
-- With Packager cloned to your local `govuk` folder, download your new package to the Packager root folder.
-- Start the VM, move to the Packager project and run `sudo dpkg -i ./your_package_name.deb`.
-- Ensure your package has been successful.
+- With Packager cloned to your local `govuk` folder, download your new package to
+the Packager root folder.
+- Start the VM, move to the Packager project and run:
 
-If successful, you can copy this package to the aptly machine and then add the deb file to aptly.
+    ```
+      $ sudo dpkg -i ./your_package_name.deb`
+    ```
+- Ensure your package has been successfully installed.
+
+If successful, you can copy this package to the aptly machine and then add the
+deb file to aptly.
+
+## Uploading a new package
```

Comment:
Opinions welcome about whether this section should be here or moved to /manual/ruby.html

Reply:
I think this is generic information for Debian packaging, so it is OK to be here? Maybe we can mention that in this example we are upgrading Ruby, so the package and repo names will change.

Reply:
👍 Added a note here https://github.com/alphagov/govuk-developer-docs/pull/1527/files#diff-1283df6a1214887a39a944667e56b29fR156